src/main/webapp/user-page.html

```diff
@@ -62,69 +62,54 @@ <h1 class="site-heading text-center text-white d-none d-lg-block">
       </div>
     </nav>
 
-    <!-- Text on background -->
-    <section class="page-section cta">
+
+    <!-- Message display -->
+    <section class="page-section cta" id="messages">
+
+      <!-- Text on background -->
       <div class="container">
         <div class="row">
           <div class="col-xl-9 mx-auto">
             <div class="cta-inner text-center rounded">
               <h2 class="section-heading mb-4">
-                <span class="section-heading-lower">Write a new post:</span>
+                <span class="section-heading-upper">Find more ...</span>
+                <span class="section-heading-lower">About Me</span>
               </h2>
+              <div class="text-center" id="about-me-container">
+                <p class="text-center" contenteditable="true" id="about-me-text">Loading...</p>
+                <button type="button" class="btn btn-warning" id="save-button" onclick="setAboutMe()">Save</button>
```

Comment:
This button as well as the profile pic text/buttons should also be hidden when viewing another user's page, similar to how the post message button is hidden.